Solution/source/Menu/Routine.cpp

```diff
@@ -157,33 +154,24 @@ void ThreadMenyooConfig()
 			{
 				MenuConfig::ConfigRead();
 			}
-			bConfigHasNotBeenRead = false;
+			g_ConfigHasNotBeenRead = false;
 		}
 
+		if (GetTickCount() > g_MenyooConfigTick + 30000)
```

Comment:
I'm not sure about this one but wouldn't `if( _programTick % 30000 )` work? Cause in that case there's no need for the g_MenyooConfigTick variable 😄

Reply:
No, `_programTick` was a local variable in the function, but since I changed the function to run each tick, it should be a global.

Reply:
Oh yeah thats right, my bad, `_programTick` was a local variable.
But yes` GetTickCount() % 30000` should work